Warning about multiple spaces in @click function name when on root element and using CRLF linebreaks

This is probably a very rare edge case related to how Windows linebreaks are interpreted. It doesn't occur when using normal Unix linebreaks (Sadly not an option on the project I'm working).

I don't expect such rare cases to be a priority or even be fixed. I just want to document it in case someone else falls into the same situation.

**Tell us about your environment**	
* **ESLint version:** 5.8.0
* **eslint-plugin-vue version:** 5.0.0
* **Node version:** 11.6.0

**Please show your full configuration:**
```json
{
  "root": true,
  "env": {
    "browser": true,
    "es6": true,
    "node": true
  },
  "extends": [
    "plugin:vue/strongly-recommended",
    "@vue/airbnb"
  ],
  "rules": {
    "max-len": "off",
    "no-param-reassign": 0,
    "linebreak-style": [
      "error",
      "windows"
    ]
  },
  "parserOptions": {
    "parser": "babel-eslint"
  }
}
```

**What did you do?**
New project from Vue CLI v3. Set up Eslint to use `plugin:vue/strongly-recommended`. Edit component as below and get warning about multiple spaces on the two first characters in the function name. 

```vue
<template>
  <div
    @click="sayHello" // Eslint warning - vue/no-multi-spaces
    class="hello"
  >
    <h1
      @click="sayHello" // No warning
      class="hello"
    >
      Hello
    </h1>
  </div>
</template>

<script>
export default {
  name: 'HelloWorld',
  methods: {
    sayHello() {
      return null;
    },
  },
};
</script>

```

**What did you expect to happen?**
After splitting attributes to their own line, there should be no errors or warnings. Using `--fix` should not do anything to the file.

**What actually happened?**
```
➜  vue-eslint git:(master) ✗ ./node_modules/.bin/eslint src/components/HelloWorld.vue

./src/components/HelloWorld.vue
  3:13  warning  Multiple spaces found before 'yHello"
'  vue/no-multi-spaces

✖ 1 problem (0 errors, 1 warning)
  0 errors and 1 warning potentially fixable with the `--fix` option.
```

Eslint starting to remove the function of `@click="sayHello"` when using VSCode Eslint fix. Doing it from terminal results in an stacktrace

```
➜  vue-eslint git:(master) ✗ ./node_modules/.bin/eslint src/components/HelloWorld.vue --fix
TypeError: Cannot read property 'range' of null
    at TokenStore.getTokenBefore (./node_modules/eslint-plugin-vue/node_modules/vue-eslint-parser/index.js:3452:102)
    at getFirstAndLastTokens (./node_modules/eslint-plugin-vue/lib/utils/indent-common.js:364:28)
    at processNodeList (./node_modules/eslint-plugin-vue/lib/utils/indent-common.js:400:31)
    at EventEmitter.VOnExpression (./node_modules/eslint-plugin-vue/lib/utils/indent-common.js:970:7)
    at EventEmitter.emit (events.js:188:13)
    at NodeEventGenerator.applySelector (./node_modules/eslint-plugin-vue/node_modules/vue-eslint-parser/index.js:3077:26)
    at NodeEventGenerator.applySelectors (./node_modules/eslint-plugin-vue/node_modules/vue-eslint-parser/index.js:3091:22)
    at NodeEventGenerator.enterNode (./node_modules/eslint-plugin-vue/node_modules/vue-eslint-parser/index.js:3099:14)
    at traverse (./node_modules/eslint-plugin-vue/node_modules/vue-eslint-parser/index.js:110:13)
    at traverse (./node_modules/eslint-plugin-vue/node_modules/vue-eslint-parser/index.js:122:13)
```

It does only happen when the `@click` is located on the root element. Also, switching the Eslint configuration to use `plugin:vue/recommended` still give the warning but how the order fixes are executed, it never triggers.

--- a/README.md
+++ b/README.md
@@ -40,7 +40,7 @@ Since single file components in Vue are not plain JavaScript, the default parser
 
 To know more about certain nodes in produced ASTs, go here:
 - [ESTree docs](https://github.com/estree/estree)
-- [vue-eslint-parser AST docs](https://github.com/mysticatea/vue-eslint-parser/blob/master/docs/ast.md)
+- [vue-eslint-parser AST docs](https://github.com/vuejs/vue-eslint-parser/blob/master/docs/ast.md)
 
 The `vue-eslint-parser` provides a few useful parser services that help traverse the produced AST and access tokens of the template:
 - `context.parserServices.defineTemplateBodyVisitor(visitor, scriptVisitor)`

--- a/docs/developer-guide/README.md
+++ b/docs/developer-guide/README.md
@@ -39,7 +39,7 @@ Since single file components in Vue are not plain JavaScript, we can't use the d
 
 To know more about certain nodes in produced ASTs, go here:
 - [ESTree docs](https://github.com/estree/estree)
-- [vue-eslint-parser AST docs](https://github.com/mysticatea/vue-eslint-parser/blob/master/docs/ast.md)
+- [vue-eslint-parser AST docs](https://github.com/vuejs/vue-eslint-parser/blob/master/docs/ast.md)
 
 The `vue-eslint-parser` provides few useful parser services, to help traverse the produced AST and access tokens of the template:
 - `context.parserServices.defineTemplateBodyVisitor(visitor, scriptVisitor)`

--- a/docs/rules/html-indent.md
+++ b/docs/rules/html-indent.md
@@ -82,7 +82,7 @@ This rule enforces a consistent indentation style in `<template>`. The default s
   - `closeBracket.endTag` (`integer`) ... The multiplier of indentation for right brackets of end tags (`</div>`). Default is `0`.
   - `closeBracket.selfClosingTag` (`integer`) ... The multiplier of indentation for right brackets of start tags (`<div/>`). Default is `0`.
 - `alignAttributesVertically` (`boolean`) ... Condition for whether attributes should be vertically aligned to the first attribute in multiline case or not. Default is `true`
-- `ignores` (`string[]`) ... The selector to ignore nodes. The AST spec is [here](https://github.com/mysticatea/vue-eslint-parser/blob/master/docs/ast.md). You can use [esquery](https://github.com/estools/esquery#readme) to select nodes. Default is an empty array.
+- `ignores` (`string[]`) ... The selector to ignore nodes. The AST spec is [here](https://github.com/vuejs/vue-eslint-parser/blob/master/docs/ast.md). You can use [esquery](https://github.com/estools/esquery#readme) to select nodes. Default is an empty array.
 
 ### `2, {"attribute": 1, "closeBracket": 1}`
 

--- a/docs/rules/no-restricted-syntax.md
+++ b/docs/rules/no-restricted-syntax.md
@@ -46,8 +46,8 @@ Forbids call expressions inside mustache interpolation.
 
 [no-restricted-syntax]: https://eslint.org/docs/rules/no-restricted-syntax
 [ESTree]: https://github.com/estree/estree
-[vue-eslint-parser]: https://github.com/mysticatea/vue-eslint-parser
-[vue-eslint-parser - AST docs]: https://github.com/mysticatea/vue-eslint-parser/blob/master/docs/ast.md
+[vue-eslint-parser]: https://github.com/vuejs/vue-eslint-parser
+[vue-eslint-parser - AST docs]: https://github.com/vuejs/vue-eslint-parser/blob/master/docs/ast.md
 
 ## :mag: Implementation
 

--- a/docs/rules/script-indent.md
+++ b/docs/rules/script-indent.md
@@ -59,7 +59,7 @@ This rule has some options.
 - `TYPE` (`number | "tab"`) ... The type of indentation. Default is `2`. If this is a number, it's the number of spaces for one indent. If this is `"tab"`, it uses one tab for one indent.
 - `baseIndent` (`integer`) ... The multiplier of indentation for top-level statements. Default is `0`.
 - `switchCase` (`integer`) ... The multiplier of indentation for `case`/`default` clauses. Default is `0`.
-- `ignores` (`string[]`) ... The selector to ignore nodes. The AST spec is [here](https://github.com/mysticatea/vue-eslint-parser/blob/master/docs/ast.md). You can use [esquery](https://github.com/estools/esquery#readme) to select nodes. Default is an empty array.
+- `ignores` (`string[]`) ... The selector to ignore nodes. The AST spec is [here](https://github.com/vuejs/vue-eslint-parser/blob/master/docs/ast.md). You can use [esquery](https://github.com/estools/esquery#readme) to select nodes. Default is an empty array.
 
 ::: warning Note
 This rule only checks `.vue` files and does not interfere with other `.js` files. Unfortunately the default `indent` rule when turned on will try to lint both, so in order to make them complementary you can use `overrides` setting and disable `indent` rule on `.vue` files:

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
     "eslint-utils": "^2.1.0",
     "natural-compare": "^1.4.0",
     "semver": "^7.3.2",
-    "vue-eslint-parser": "^7.1.0"
+    "vue-eslint-parser": "^7.1.1"
   },
   "devDependencies": {
     "@types/eslint": "^7.2.0",

--- a/tests/lib/rules/v-on-function-call.js
+++ b/tests/lib/rules/v-on-function-call.js
@@ -117,7 +117,7 @@ tester.run('v-on-function-call', rule, {
     {
       filename: 'test.vue',
       code: '<template><div @click="foo"></div></template>',
-      output: `<template><div @click="foo"></div></template>`,
+      output: null,
       errors: [
         "Method calls inside of 'v-on' directives must have parentheses."
       ],
@@ -242,6 +242,15 @@ tester.run('v-on-function-call', rule, {
       <template>
         <div @click="fn"></div>
       </template>`,
+      errors: [
+        "Method calls without arguments inside of 'v-on' directives must not have parentheses."
+      ],
+      options: ['never']
+    },
+    {
+      filename: 'test.vue',
+      code: '<template>\r\n<div\r\n@click="foo()" /></template>',
+      output: '<template>\r\n<div\r\n@click="foo" /></template>',
       errors: [
         "Method calls without arguments inside of 'v-on' directives must not have parentheses."
       ],

--- a/typings/eslint-plugin-vue/util-types/ast/v-ast.ts
+++ b/typings/eslint-plugin-vue/util-types/ast/v-ast.ts
@@ -1,5 +1,5 @@
 /**
- * @see https://github.com/mysticatea/vue-eslint-parser/blob/master/docs/ast.md
+ * @see https://github.com/vuejs/vue-eslint-parser/blob/master/docs/ast.md
  */
 import { HasParentNode, BaseNode } from '../node'
 import { Token, HTMLComment, HTMLBogusComment, Comment } from '../node'

